pupil-api/src/App.ts

```diff
@@ -9,6 +9,8 @@ import * as helmet from 'helmet'
 import * as uuidV4 from 'uuid/v4'
 import * as winston from 'winston'
 import * as azure from './azure'
+import { rateLimit } from './helpers/rate-limit'
```

Comment:
@jon-shipley where is this file? is it pushed to the remote branch?
Can't seem to find it and from my understanding it's a new one

Reply:
Oops! 🤦‍♂️ 

I've added it now.  Thank you!